src/Microsoft.IdentityModel.Tokens/opensource/json/DefaultJsonNameTable.cs

```diff
@@ -77,7 +80,12 @@ public override string Get(char[] key, int start, int length)
             hashCode -= hashCode >> 5;
 
             // make sure index is evaluated before accessing _entries, otherwise potential race condition causing IndexOutOfRangeException
-            var index = hashCode & _mask;
+#if NET20 || NET35 || NET40
```

Comment:
>NET35 || NET40 [](http://example.com/codeflow?start=13&length=14)

do we need these?

Reply:
We don't need any of those defines

Reply:
Both are in the System.Threading and apply to all platforms we support.